### DISCUSSION TITLE
Reason for error: failed to connect to all addresses

### DISCUSSION THREAD

**Discussion by etiennelame** [ansys/pyfluent#3597] [General]
I recently using pyfluent for calculation. The case worked well at the begining, but once the calculation timestep reached 722, the process broke and the python console show error of **failed to connect to all addresses**. At the beginning, I tried some method with ip address and tried reinstall the ANSYS, it did not work. 

And then, I checked my problem case and found that it was ridiculously slow, even I replace with a mesh of 80000 cell, it was still ridiculously slow. But if I create a new case with the 80000-cell mesh, it can solve quickly.  

Accidently, I write out the setting file with TUI command **file/w-s resi**. In the setting file, I found  numberous of data was recorded in the rp-variables **residuals/settings-transient** and **residuals/settings**. After deleting the data of these two rp-variables in setting file, and re-read the setting file, the problem case worked well again. 

I think that the reason for the problem is that: the case kept storing data into the rp-variables **residuals/settings-transient** and **residuals/settings**, too much data can slow down the process and maybe raise some error in Fluent and shut it down, then the pyfluent show the error code. 

This is the setting file with problem. If anyone had idea to avoid this problem, please share the method and I appreciate your kindness
[case12c_2.txt](https://github.com/user-attachments/files/18217935/case12c_2.txt)
.

**Comment by ansjschuetz** [comment]
Hello Etienne,
congratulations on the detailed analysis!
There has been a known issue (not specific to PyFluent) with data in those RP variables growing continuously under unusual circumstances.
It should not be reproducible with the most recent available release 2024 R2.
Can you perhaps consider upgrading your Ansys software?
Kind regards,
Jochen
